subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/internal/RuntimeApplicationDirProvider.kt

```diff
@@ -0,0 +1,39 @@
+package com.avito.android.internal
+
+import android.content.Context
+import android.os.Build
+import androidx.core.content.ContextCompat
+import com.avito.report.ApplicationDirProvider
+import com.avito.report.ApplicationDirProviderFactory
+import java.io.File
+
+internal class RuntimeApplicationDirProvider(context: Context) : ApplicationDirProvider {
```

Comment:
Move to test-instrumentation-runner

Reply:
Can't place it inside `test-instrumentation-runner`.
Now, this module looks like a generic abstraction over AndroidJunitRunner which doesn't know about test runner.
Let it be that way.

Reply:
okay, time to extract another part from inhouse runner then ) )